docs/current_docs/cookbook/cookbook.mdx

```diff
@@ -1505,9 +2409,23 @@ The following Dagger Function demonstrates how to use native-language concurrenc
 
 Execute a Dagger Function which performs different types of tests by executing other Dagger Functions concurrently.
 
-```shell
-dagger call --source=. run-all-tests
-```
+    <Tabs groupId="shell">
```

Comment:
Ping @helderco @jasonmccallister for double-check on this example

Reply:
You need `host` if you need to go outside the current module's context. If you start with `-n` it works like `dagger call` though. You could use some other path within the context but then there wouldn't be an example on reaching for a directory outside of the repo. 

So to avoid confusion I think it's good to keep `$(host)`, but should update the path in the `call` command as well.

Reply:
@helderco I've updated this but I think the `call` command is good as is? I didn't understand why you suggested updating that path.

Reply:
> I didn't understand why you suggested updating that path.

Oh, I think that was meant for something else, in "Set a module-wide default path", dagger call uses `/src/myapp` while shell uses `/home/admin/images`.

Reply:
Fixed in https://github.com/dagger/dagger/pull/9936